libs/auth-api-lib/src/lib/clients/dto/base/client-base.dto.ts

```diff
@@ -314,4 +314,11 @@ export abstract class ClientBaseDTO {
     example: true,
   })
   readonly promptDelegations!: boolean
+
+  @IsBoolean()
+  @IsNotEmpty()
```

Comment:
`@IsNotEmpty` is unnecessary check as IsBoolean will assert a boolean value.

Reply:
Removed.